kouta-backend/src/main/scala/fi/oph/kouta/util/NameHelper.scala

```diff
@@ -110,4 +111,17 @@ object NameHelper {
       hakukohdeNimi
     }
   }
+
+  def generateMuokkaajanNimi(henkilo: Henkilo): String = {
+    val kutsumanimi = henkilo.kutsumanimi.getOrElse("")
+    val etunimet = henkilo.etunimet.getOrElse("")
+    val lastname = henkilo.sukunimi.getOrElse("")
+
+    var firstname = kutsumanimi
+    if (firstname.isEmpty) {
+      firstname = etunimet
+    }
```

Comment:
Muuttujaa asettaessa voi iffailla suoraan näin
```suggestion
       val firstname = if (kutsumanimi.nonEmpty) kutsumanimi else etunimet
```

Reply:
Muutin tän kun toi ternary on siistimmän näköinen.